### DISCUSSION TITLE
Update list that has a choice field value using pwsh

### DISCUSSION THREAD

**Discussion by DII-dsward** [pnp/powershell#465] [Q&A] [answered]
Need help updating a drop down field (type: choice) in our sharepoint list to a specific value. These values below are all working except EnvName which needs to be set to Prod or Non-Prod.

Set-PnPListItem -List $ticlistid -ContentType "$($Title)" -Identity $hostid -Values @{
        "Title"                                          = "$($Title)"; 
        "AzureResourceGroup"               = "$($AzResourceGroup)";
        "Physical_x0020_Location"          = "$($PhysicalLocation)";
        "EnvName"                                  = "$($EnvName)";
        }

**Comment by veronicageek** [comment]
Is there an error message or something?

**Reply by DII-dsward** [reply]
I wasn't getting an error, it just wasn't writing a value. I think I found the cause of the issue to be how I'm setting the EnvName variable. I tried plain text and it says it writes but when i go to the sharepoint field i didn't see a value written.

**Comment by DII-dsward** [comment, marked as answer]
Issue found. Was not a PnP issue. Turned out the way I'm gathering the EnvName variable was incorrect.
